provider/cmd/app/app.go

```diff
@@ -88,6 +92,18 @@ type Options struct {
 	ResyncIntervalGarbageCollector time.Duration
 }
 
+type HTTPServerOptions struct {
+	Addr            string
+	ReadTimeout     time.Duration
+	WriteTimeout    time.Duration
+	IdleTimeout     time.Duration
```

Comment:
Can we go simple here? I propose to extend the options if we see any issues and drop it for now.

Reply:
why? it is configurable, you can redefine it, if you need it or use default value. But you have this option and you don't have to wait for implementation. It can be helpful because you don't want to have multiple long living connection because you have limited count of filedescriptors which can be open at the time. So indirectly connection can affects main logic.

Reply:
fixed